### PR TITLE
Update Readme for WEBTS Repository

### PR DESCRIPTION
**Pull request by streetcoder123**
**Changes**
- Necessary updates to the WebTS repository based on changes on adding Azure App Services and moving Azure functions back to preview mode.
- Also updates the screenshot for the Example Scenario.

New Screenshot:
image

### CODE REVIEW

**Review by annielovesjs — DISMISSED** [review]

**Review by jasmingoh — APPROVED** [review]

**Review by jsondoo — APPROVED** [review]

**Review by ali-kha — APPROVED** [review]

**Review by annielovesjs — APPROVED** [review]

**Review by jasmingoh — CHANGES_REQUESTED** [review]
Waiting for @xtinah-w 's approval

**Review by jasmingoh — APPROVED** [review]

**Pull request merged**
